sale_stock_picking_blocking/__manifest__.py

```diff
@@ -9,14 +9,18 @@
     "author": "Eficent, Odoo Community Association (OCA)",
```

Comment:
Could you take the ocassion to rename Eficent to Forgeflow in all the ocurrences?